include/emqx_ct_helpers.hrl

```diff
@@ -0,0 +1,140 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2020 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-define( MQTT_SSL_TWOWAY, [ { cacertfile, filename:join( [ "etc", "certs", "cacert.pem" ] ) },
+							{ verify, verify_peer },
+							{ fail_if_no_peer_cert, true } ] ).
+
+-define( MQTT_SSL_CLIENT_CERTS, [ { keyfile,	filename:join( [ "etc", "certs", "client-key.pem" ]) },
```

Comment:
line too long. please keep width within 100 chars.

Reply:
Refactored in update